site/content/docs/5.2/examples/features/index.html

```diff
@@ -285,4 +285,63 @@ <h4 class="fw-bold mb-0">Featured title</h4>
       </div>
     </div>
   </div>
+
+  <div class="b-example-divider"></div>
+
+  <div class="container px-4 py-5">
+    <h2 class="pb-2 border-bottom">Features with Title</h2>
+
+    <div class="row row-cols-1 row-cols-md-2 align-items-md-center g-4 py-5">
+        <div class="d-flex flex-column align-items-start gap-2">
+            <h2 class="fw-bold">
+                Left Aligned Title explaining the Awesome Features
+            </h2>
+            <p class="text-muted">
+                Paragraph of text beneath the heading to explain the heading. We'll add onto it with another sentence and probably just keep going until we run out of words.
+            </p>
+            <a href="#" class="btn btn-primary btn-lg">Primary</a>
```

Comment:
Nitpick: expand the text in the button for more accurate real world use.

```suggestion
            <a href="#" class="btn btn-primary btn-lg">Primary button</a>
```